src/main/java/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,17 @@
+package dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
```

Comment:
The class begins with an empty line. According to the checklist, do not begin class or method implementation with an empty line. Please remove the redundant empty line.